pkg/utils/packet.go

```diff
@@ -175,26 +176,27 @@ func AnnounceIPs(ifName string, ipConfigs []*current.IPConfig) error {
 		return fmt.Errorf("invalid Ethernet MAC address: %q", linkObj.Attrs().HardwareAddr)
 	}
 
+	var errResult error
```

Comment:
nit: can you document this new behaviour in AnnounceIPs docstring ?

Reply:
done. How is this?